export_report.py

```diff
@@ -0,0 +1,156 @@
+#!/usr/bin/python3
+
+import plotly.io as pio
+import pandas as pd
+from docx import Document
+from docx.shared import Inches
+import os
+import re
+import sys
+import datetime
+
+def generateGraphs():
```

Comment:
Do you mind to use the Python style naming conventions? (e.g. `generate_graphs()`)